pym/bob/generators/QtCreatorGenerator.py

```diff
@@ -200,11 +206,11 @@ def qtProjectGenerator(package, argv, extra):
     getCheckOutDirs(package, excludes, dirs)
     if not projectName:
         # use package name for project name
-        projectName = package.getName()
+        projectName = package.getName().replace('::', '__')
```

Comment:
Replace "::" is important in some case, because of "C:/" shall not match and replaced to "C_/".